Datasets/Sentinel_1.ipynb

```diff
@@ -0,0 +1,655 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sentinel 1 <img align=\"right\" src=\"../Supplementary_data/DE_Africa_Logo_Stacked_RGB_small.jpg\">\n",
+    "\n",
+    "* **Products used:**\n",
+    "[sentinel1_ghana_monthly](https://explorer.digitalearth.africa/sentinel1_ghana_monthly)\n",
+    "<!-- [Level-1 Single Look Complex and Ground Range Detected](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-1), -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background\n",
+    "\n",
+    "The [Sentinel-1](https://en.wikipedia.org/wiki/Sentinel-1) mission is composed of a constellation of two Synthetic Aperture Radar (SAR) satellites, Sentinel-1A and Sentinel-1B, sharing the same orbital plane. Sentinel-1 provides dual polarisation capability, very short revisit times, and rapid product delivery.\n",
+    "\n",
+    "SAR data has the advantage of operating at wavelengths not impeded by cloud cover or a lack of illumination and can acquire data over a site during the day or night, and under most weather conditions. The Sentinel-1 mission is the European Radar Observatory for the Copernicus joint initiative of the European Commission (EC) and the European Space Agency (ESA) that can offer reliable and repeated wide area monitoring with its SAR instrument. \n",
+    "\n",
+    "Sentinel-1 products will benefit users interested in agriculture, forestry and land cover classification. The rapid data dissemination and short revisit cycles of Sentinel-1 together with its interferometric capabilities will also benefit emergency response users, such as the United Nations [International Charter on Space and Major Disasters](https://disasterscharter.org/web/guest/home;jsessionid=83C5354659FD4EFBE0A8D388BBA5B56F.jvm1), in emergency situations such as floods, earthquakes, volcanic eruptions and landslides.\n",
+    "\n",
+    "Sentinel-1A was launched on 3 April 2014 and Sentinel-1B followed on 25 April 2016. Two other spacecraft (Sentinel-1C and Sentinel-1D) are planned to replace the first two satellites at the end of their operational lifespan. For more information on the Sentinel-1 platforms and applications, check out the [European Space Agency](https://sentinel.esa.int/web/sentinel/missions/sentinel-1) website.\n",
+    "\n",
+    "The Normalized Radar Backscatter product needs Radiometric Terrain Correction (RTC) that is called gamma0 (γ0) backscatter. In this product the systematic contamination that would otherwise still be present in sets of data acquired with multiple geometries will be mitigated.\n"
+   ]
```

Comment:
Digital Earth Africa applies corrections to Sentinel-1 GRD images to arrive at a Terrain-flattened gamma nought product (see the introduction to Digital Earth Australia section for more information on the Terrain-flattened gamma nought data). - Please remove the statement in the parenthesis. we need to add information for gamma0 in the DEA section if we decide to have this statement.